pycbc/inject/inject.py

```diff
@@ -739,9 +747,246 @@ def supported_approximants():
         return list(waveform.ringdown_td_approximants.keys())
 
 
+class FromFileHDFInjectionSet(_HDFInjectionSet):
+    """Manages injecting an arbitrary time series loaded from a file.
+
+    The injections must have the following attributes set:
+
+    * ``filename``: (str) the name of the file to load containing the time
+      series. The file type and format can be a frame file or anything
+      understood by :py:func:`pycbc.types.timeseries.load_timeseries`. If a
+      frame file (ends in ``.gwf``) is specified, a ``channel`` attribute must
+      also be set.
+
+    * ``DETECTOR_gps_time``: (float) The GPS time at which the time series
```

Comment:
This makes it look like you aren't using the standard multi ifo opt format. Can you switch to that? So in the config file it would be 
```
gps_time = H1:2323423 L1:234324234
```

We should get away from non-standard formats.

Reply:
We discussed this on the call, but just to have a record of it... the reason for the current naming is the times are read here from an hdf file, where each parameter is a dataset. Doing what is suggested here would require storing variable length strings (a potential problem if you tried to add a new time that was longer). We decided it was better to just store things as `DETECTOR_gps_time`, as it currently is.